requirements.txt

```diff
@@ -0,0 +1,21 @@
+appdirs==1.4.4
+attrs==20.3.0
+cfgv==3.2.0
+distlib==0.3.1
+filelock==3.0.12
+identify==1.5.13
+iniconfig==1.1.1
+nodeenv==1.5.0
+numpy==1.19.5
+packaging==20.8
+pluggy==0.13.1
+pre-commit==2.9.3
+py==1.10.0
+pyparsing==2.4.7
+pytest==6.2.1
+PyYAML==5.3.1
+six==1.15.0
+toml==0.10.2
+torch==1.7.1
+typing-extensions==3.7.4.3
+virtualenv==20.4.0
```

Comment:
Do we really need to specify all these dependencies?  Why not just listing the ones we actually need?

Reply:
True, maybe not for development, but for the end model we want an exact `requirements.txt` like this for reproducability reasons

Reply:
We can probably replace this with 

```
torch
pytest
```

Reply:
Changed it now

Reply:
Yeah, reproducability is important! Maybe stick with pinned versions, but only for the packages we need. That way we trade some reproducibility with less dependency conflicts later on when we add packages.
```
torch==1.7.1
pytest==6.2.1
```
WDYT?

Reply:
LGTM!